### PR TITLE
Setup mapstruct into backend

### PR DESCRIPTION
**Pull request by hanswizzy**
@stephane-segning Was tasked to implement mapstruct into the backend and write a test file to ensure everything works.
Please review!!!

### CODE REVIEW

File: power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/Person.java
Comment (stephane-segning) [inline power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/Person.java:1]: Put this class inside an inner class inside of the `MapstructTest`.

File: power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PersonDto.java
Comment (stephane-segning) [inline power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PersonDto.java:1]: Put this class inside an inner class inside of the `MapstructTest`.

File: power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PersonMapper.java
Comment (stephane-segning) [inline power-pay-backend/src/test/java/com/adorsys/gis/powerpay/powerpaybackend/PersonMapper.java:1]: Put this class inside an inner class inside of the `MapstructTest`.

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work @Elwizzy12. Please watch this

**Review by stephane-segning — APPROVED** [review]

**Review by NkwaTambe — APPROVED** [review]

**Review by Arielpetit — APPROVED** [review]

**Review by ArmandMeppa — APPROVED** [review]

**Comment by Motouom** [comment]
@Elwizzy12  please look for why your ticket can't be merged and fix it

**Pull request closed without merging**